request_handlers_test.cc

```diff
@@ -4,289 +4,213 @@
 #include "request_handlers.h"
 #include "request_router.h"
 #include "server_config.h"
+#include "request.h"
+#include "response.h"
 
 #include <cstddef>
 #include <fstream>
 #include <sstream>
 
 class RequestHandlersTest : public ::testing::Test {
 protected:
+  NginxConfig config;
+  NginxConfigParser config_parser;
+  ServerConfig server_config;
```

Comment:
Just wondering, are we following convention to add a `_` at the end for protected class variables?